### DISCUSSION TITLE
Emby外部播放器用mpv看，设备为笔记本外接显示器，鼠标移动窗口框出来后缓存就清空了

### DISCUSSION THREAD

**Discussion by wonderhj** [diana7127/mpv.net-DW#38] [Q&A]
在6秒左右，鼠标移动到窗口框那里，readahead就清空了，要重新缓存。

https://github.com/diana7127/mpv.net-DW/assets/23394989/6b7249a6-fcca-47ca-9478-c2aa19adb25f

**Comment by witherleaves** [comment]
解决了吗 我也是这样子，在网上也没找到相关解决方法，求指导

**Reply by witherleaves** [reply]
找到了个解决方法在F:\MPV_NET\mpv.net-DW v2.0.0\portable_config\scripts文件夹里面移除modernx.lua和thumbfast.lua 再在mpv config文件里面修改osc=yes 基本上就没问题了就是进度条变得简陋了一点功能没啥毛病，然后可以把缓存改大一点，方便跳转
